src/main/java/core/basesyntax/Random.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Random {
```

Comment:
I don't think we need this class, we can use java.util.Random created once as a field in FigureSupplier. But if you have this class, avoid recreating java.util.Random instance per each method invocation, it should have been a field